02-household-queries/dspy_engine.py

```diff
@@ -188,7 +188,10 @@ def create_llm_model(llm_name="openhermes"):
         "gpt-4",
         "gpt-4-turbo",
     ]:
-        return dspy.OpenAI(model=llm_name, temperature=0.1, response_format={"type": "json_object"})
+        if respond_with_json:
```

Comment:
should there be an option to set the temperature in the param?

Reply:
There could be but I haven't needed to change the temperature yet.